### ISSUE TITLE
Build failure in spicy-driver

### ISSUE THREAD

**Issue by 0xxon** [zeek/zeek#2553]
Hi,

OBS recently encountered a build-failure that I have never seen like this before.

Important log part:

```
[ 5727s] [ 97%] Building CXX object src/CMakeFiles/zeek_objs.dir/script_opt/ZAM/Vars.cc.o
[ 5730s] [ 97%] Building CXX object src/CMakeFiles/zeek_objs.dir/script_opt/ZAM/ZBody.cc.o
[ 5825s] [ 97%] Building CXX object src/CMakeFiles/zeek_objs.dir/script_opt/ZAM/ZInst.cc.o
[ 5827s] [ 98%] Building CXX object src/CMakeFiles/zeek_objs.dir/script_opt/ZAM/ZOp.cc.o
[ 5831s] make[4]: Leaving directory '/usr/src/packages/BUILD/build'
[ 5831s] [ 98%] Built target zeek_objs
[ 5831s] make[4]: Entering directory '/usr/src/packages/BUILD/build'
[ 5831s] Scanning dependencies of target zeek
[ 5831s] make[4]: Leaving directory '/usr/src/packages/BUILD/build'
[ 5831s] make[4]: Entering directory '/usr/src/packages/BUILD/build'
[ 5831s] [ 98%] Building CXX object src/CMakeFiles/zeek.dir/main.cc.o
[ 5834s] make[4]: *** No rule to make target 'src/builtin-plugins/spicy-plugin/src/compiler/CMakeFiles/zeek-compiler.dir/driver.cc.o', needed by 'src/zeek'.  Stop.
[ 5834s] make[4]: Leaving directory '/usr/src/packages/BUILD/build'
[ 5834s] make[3]: *** [CMakeFiles/Makefile2:6633: src/CMakeFiles/zeek.dir/all] Error 2
[ 5834s] make[3]: Leaving directory '/usr/src/packages/BUILD/build'
[ 5834s] make[2]: *** [Makefile:182: all] Error 2
[ 5834s] make[2]: Leaving directory '/usr/src/packages/BUILD/build'
[ 5834s] make[1]: *** [Makefile:15: all] Error 2
[ 5834s] make[1]: Leaving directory '/usr/src/packages/BUILD'
[ 5834s] make: *** [debian/rules:45: build-stamp] Error 2
[ 5834s] dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

Full log in attachment. So far this happened on Debian 11, Ubuntu 22.04, and Fedora on several architectures.

@bbannier mentioned that this might have something to do with some recent work by @sethhall, and might be a race-condition.

[debian11.log](https://github.com/zeek/zeek/files/9971406/debian11.log)

**Comment by 0xxon** [comment]
Title changed since the failure started appearing in more and more systems.

**Comment by bbannier** [comment]
This will be fixed by the patch for #2586, closing this one.
